src/features/cosoul/CoSoulManagement.tsx

```diff
@@ -32,14 +39,33 @@ export const CoSoulOverview = ({
       <Flex column css={{ gap: '$sm' }}>
         <Text variant="label">{"You've Earned"}</Text>
         <Text h2 display>
-          {numberWithCommas(cosoul_data.totalPgive, 0)} Public GIVE
+          <Text
+            color="cta"
+            className={`${!mintInfo && 'management-scramble'}`}
```

Comment:
this seems confusing to me. you're wanting to set the classname only if mintInfo is falsey, maybe do it like this:

`
className={!mintInfo ? 'management-scramble' : ''}
`